test/controllers/api/v1/tags_test.rb

```diff
@@ -0,0 +1,103 @@
+# == Schema Information
+#
+# Table name: tags
+#
+#  id                 :integer          not null, primary key
+#  name               :string
+#  description        :text
+#  color              :string
+#
+
+require 'test_helper'
+
+class API::V1::TagsTest < ActiveSupport::TestCase
+  include Rack::Test::Methods
+
+  def app
+    Rails.application
+  end
+
+  setup do
+    set_default_settings
+    @user = users(:admin)
+    @api_key = ApiKey.create(name: "Test Tags Key", user: @user)
+    @default_params = { token: @api_key.access_token }
+
+    @simple_tag = ActsAsTaggableOn::Tag.new(name: 'simple', description: nil, color: nil)
+    @full_tag = ActsAsTaggableOn::Tag.new(name: 'full', description: 'description', color: '#336699')
```

Comment:
During my tests, there were no tags in the db. If you wish, I can try to move this setup into fixture files.

Reply:
I think this is fine